src/App.tsx

```diff
@@ -1,223 +1,200 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
+import { TodosFilter, TodoError } from './types/TodoFilter';
+import { ErrorInfo } from './components/ErrorInfo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [formInput, setFormInput] = useState('');
+  const [filter, setFilter] = useState<TodosFilter>(TodosFilter.all);
+  const [error, setError] = useState<TodoError>(TodoError.noerror);
+  const [isAdding, setIsAdding] = useState(false);
+
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const getTodosFromServer = () => {
+    getTodos(user?.id)
+      .then(setTodos);
+  };
+
+  const todosFiltration = (filterType: TodosFilter) => {
+    switch (filterType) {
+      case TodosFilter.active:
+        setVisibleTodos(todos.filter(todo => todo.completed === false));
+        setFilter(TodosFilter.active);
+        break;
```

Comment:
add empty line after break. Fix it all over the file

```suggestion
        break;
        
```

Reply:
Why didn't you fix it?